api-schema/src/main/java/org/stellar/anchor/api/sep/sep31/Sep31PostTransactionRequest.java

```diff
@@ -34,9 +34,13 @@ public class Sep31PostTransactionRequest {
   @SerializedName("receiver_id")
   String receiverId;
 
-  Sep31TxnFields fields;
+  @Deprecated Sep31TxnFields fields;
```

Comment:
This is going into 3.0, right? Can we remove the field instead?

Reply:
Yes this is going to 3.0 but no we cannot remove it. I tried to do so in the Protocol change but Jake insisit that we should keep it and continue support, and so far there isn't a plan to get rid of it :/

Reply:
I believe there are a number of changes going into 3.0 that are deprecated in the protocol but we decided to remove in 3.0. wdyt @lijamie98

Reply:
While I’d love to remove all those long-deprecated fields, I don’t think we can get everyone on the same page right now with the 3.0 release so close.

A more practical approach might be to create a plan after the 3.0 release, outlining the removal timeline, the specific fields to be deprecated, and a guide for identifying their replacements or alternatives.

Reply:
Sounds good. Let's merge this change as is and discuss whether we want to remove these fields in 3.x or 4.0.

Reply:
tracking in https://stellarorg.atlassian.net/browse/ANCHOR-888